README.md

```diff
@@ -1,8 +1,50 @@
 
-# plastic-packaging-tax-returns-frontend
+# Plastic Packaging Tax Returns (PPT)
 
-This is a placeholder README.md for a new repository
+This is the Scala microservice responsible for the transient storage of PPT returns information and PPT Account section,
+which is part of the PPT tax regime, as discussed in this [GovUk Guindance](https://www.gov.uk/government/publications/introduction-of-plastic-packaging-tax/plastic-packaging-tax)
```

Comment:
Guindance?

Reply:
You know from The Government Guindance. Give it a few decades, it will catch up :stuck_out_tongue: eventually

Reply:
This has been addressed.